tests/testUtils.js

```diff
@@ -84,5 +96,10 @@ module.exports = {
     .then(() => done())
     .catch((err) => done(err));
   }, // forceDeleteUser
-
+  // delete users
+  forceDeleteSubject(done) {
+    forceDelete(db.Subject, testStartTime)
+    .then(() => done())
+    .catch((err) => done(err));
+  }, // forceDeleteUser
 }; // exports
```

Comment:
the tests to delete subjects are in utils.js. There is a utils.js file under each of the folders in /tests